src/sentry/migrations/0549_migrate_no_action_dupe_issue_alerts.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 3.2.20 on 2023-09-11 19:11
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def has_duplicate_rule(apps, rule_data, project, rule_id):
+    Rule = apps.get_model("sentry", "Rule")
+
+    matchers = {key for key in list(rule_data.keys()) if key not in ("name", "user_id")}
+    extra_fields = ["actions", "environment"]
+    matchers.update(extra_fields)
+    existing_rules = Rule.objects.exclude(id=rule_id).filter(project=project, status=0)
```

Comment:
```suggestion
    existing_rules = Rule.objects.exclude(id=rule_id).filter(project=project, status=ObjectStatus.ACTIVE)
```

Reply:
I intentionally did this in case when someone runs it in the future that enum isn't available - I think in the past like [here](https://github.com/getsentry/sentry/blob/master/src/sentry/migrations/0507_delete_pending_deletion_rules.py) I copied it over to avoid that - I'll update it here.

Reply:
ahh got it -- could be worth just adding a comment. that makes sense!